config/environments/development.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
```

Comment:
Should the magic frozen string comment be removed?

Reply:
These are changes coming from Rails proper, installed by the `app:upgrade` tasks. In order to ease on upgrades it's probably best to have as few changes in these files as possible, although I am not certain of the best upgrade path.

Rubocop accepts the removal, but that's because we've told it to ignore these files (as Rails coding style doesn't quite match ours).

So... I don't know?

Reply:
Actually, I do. These are our files, we should take ownership over them even if Rails is under the delusion that it can modify them at will during `app:upgrade`.